src/features/filter.ts

```diff
@@ -1,10 +1,36 @@
-export const actions = { /* put action creators here */};
+import { Status } from '../types/Status';
 
-const filterReducer = () => {
-  return {
-    query: '',
-    status: 'all',
-  };
+type SetQueryAction = { type: 'filter/SET_QUARY', payload: string };
+type SetStatusAction = { type: 'filter/SET_STATUS', payload: Status };
+type Action = SetQueryAction | SetStatusAction;
+type Filter = { query: string, status: string };
+
+const setStatus = (status: Status): SetStatusAction => ({
+  type: 'filter/SET_STATUS',
+  payload: status,
+});
+
+const setQuery = (query: string): SetQueryAction => ({
+  type: 'filter/SET_QUARY',
```

Comment:
also move all text values to const
```suggestion
  type: 'filter/SET_QUERY',
```

Reply:
not fixed
![image](https://user-images.githubusercontent.com/98764641/227869752-0ad6f70f-b5ce-4059-ab42-a5edb85e649a.png)